web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -68,82 +69,112 @@ describe("<SidebarCardsList />", () => {
   });
 
-  it("fire unhide_cards analytics when accordion is opened when displayCertifications is true", () => {
-    render(
-      <SidebarCardsList
-        topCards={[]}
-        bottomCards={[]}
-        fundings={[]}
-        hiddenFundings={[]}
-        certifications={[]}
-        hiddenCertifications={[]}
-        displayFundings={false}
-        displayCertifications={true}
-      />
-    );
+  it("fires unhide_cards analytics when accordion is opened when displayCertifications is true", () => {
+    renderComponent({ displayCertifications: true });
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     expect(mockAnalytics.event.for_you_card_unhide_button.click.unhide_cards).toHaveBeenCalledTimes(1);
   });
 
-  describe("Empty State Messaging", () => {
-    it("displays Complete Required Tasks Message when there are no top or bottom cards and display Fundings and Certifications are false", () => {
+  describe("Empty State Messages", () => {
+    it("displays the Remote Seller/Worker specific message when the business is a Remote Seller/Worker", () => {
+      const mockBusiness = generateBusiness({
+        profileData: generateProfileData({
+          operatingPhase: "REMOTE_SELLER_WORKER",
+        }),
+      });
+      useMockBusiness(mockBusiness);
+
       renderComponent({});
-      expect(screen.getByTestId("complete-required-tasks-msg")).toBeInTheDocument();
-      expect(screen.queryByText(Config.dashboardDefaults.emptyOpportunitiesText)).not.toBeInTheDocument();
+      expect(
+        screen.getByText(Config.dashboardDefaults.emptyOpportunitiesRemoteSellerWorkerText)
+      ).toBeInTheDocument();
     });
 
-    it("does not display Complete Required Tasks Message when displayFundings is true", () => {
-      renderComponent({ displayFundings: true });
-      expect(screen.queryByTestId("complete-required-tasks-msg")).not.toBeInTheDocument();
-    });
+    describe("Empty Opportunities Message", () => {
+      it("displays when there are no visible fundings or certifications", () => {
+        renderComponent({
+          fundings: [],
+          certifications: [],
+          displayCertifications: true,
+          displayFundings: true,
+        });
 
-    it("does not display Complete Required Tasks Message when displayCertifications is true", () => {
-      renderComponent({ displayCertifications: true });
-      expect(screen.queryByTestId("complete-required-tasks-msg")).not.toBeInTheDocument();
-    });
+        const emptyMessage = screen.getByTestId("empty-for-you-message");
+        const sanitizedString = removeMarkdownFormatting(Config.dashboardDefaults.emptyOpportunitiesHeader);
+        const combinedMessage = `${sanitizedString}${Config.dashboardDefaults.emptyOpportunitiesText}`;
+        expect(emptyMessage).toHaveTextContent(combinedMessage);
+      });
```

Comment:
Instead of referencing a test id, this will sanitize any markdown formatting and allow us to get the string from the page. It still utilizes the config and is more in-line with the way we've been testing for an element's presence on the page.